chromadb/test/property/test_cross_version_persist.py

```diff
@@ -118,7 +118,7 @@ def version_settings(request) -> Generator[Tuple[str, Settings], None, None]:
     # Cleanup the persisted data
     data_path = configuration[1].persist_directory
     if os.path.exists(data_path):
-        shutil.rmtree(data_path)
+        shutil.rmtree(data_path, ignore_errors=True)
```

Comment:
context on this? would errors be that the path does not exist?

Reply:
Yeah, this is just to avoid rethrows in case of a test failure which leads to the directory not being created, in which case the error you see gets masked by this remove sometimes.